pkgs/applications/video/freetube/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, appimageTools, makeWrapper, electron }:
+
```

Comment:
i'm new here, but i think you don't need to refer to `stdenv` anymore explicitly. there was a recent repo-wide commit that removed it: https://github.com/NixOS/nixpkgs/pull/115075

Reply:
It's needed here because `stdenv.mkDerivation` is called.

Reply:
Looks like no compilation is taking place, so it could even be `stdenvNoCC`, but we should prefer a source based package.